contracts/GuestList.vy

```diff
@@ -51,6 +51,25 @@ interface Bancor:
     def protectedLiquidity(_id: uint256) -> ProtectedLiquidity: view
 
 
+event GuestInvited:
+    guest: address
+    invited: bool
```

Comment:
what would `GuestInvited` with `invited == False` mean?
Shouldn't it be `GuestInvited` / ``GuestRemoved`?

Reply:
Yes, it would mean a guest is removed. I'll change to that.